desktop_version/src/Game.cpp

```diff
@@ -442,6 +442,27 @@ void Game::updatecustomlevelstats(std::string clevel, int cscore)
     savecustomlevelstats();
 }
 
+#define LOAD_ARRAY_RENAME(ARRAY_NAME, DEST) \
+    if (pKey == #ARRAY_NAME && pText[0] != '\0') \
+    { \
+        char buffer[16]; \
```

Comment:
Could use a comment to explain why 16 is used

Reply:
Done. Rebased and force-pushed.